apps/backend/src/super-token-accounting/super-token-accounting.service.ts

```diff
@@ -53,17 +53,19 @@ export class SuperTokenAccountingService {
     if (error) {
       throw error;
     } else {
-      const streamPeriods = response.data.filter(
-        (x) => x.token!.id!.toLowerCase() === superTokenAddress.toLowerCase(),
-      );
-      const sumTransferred = streamPeriods.reduce(
-        (acc, { sender, totalAmountStreamed }) =>
-          acc +
-          (sender.toLowerCase() === senderAddress.toLowerCase() ? 1n : -1n) *
-            (totalAmountStreamed ? BigInt(totalAmountStreamed) : 0n),
-        0n,
-      );
-      return sumTransferred;
+      // const streamPeriods = response.data.filter(
```

Comment:
i commented this out for now, it produces an error that `filter` is undefined

Reply:
Right, this was not correctly typed in the Accounting API's spec, had to make the change in the spec manually.